### ISSUE TITLE
Tifffile install reports it fails, upon restart Blender finds the dependency

### ISSUE THREAD

**Issue by oanegros** [oanegros/MicroscopyNodes#43]
This happens in approx 5% of machines.

**Comment by JB4Jaison** [comment]
Thanks for this update. It actually failed on my machine - 

- Windows 11 Enterprise
- 10.0.22631 Build 22631
- Blender 4.1

The log files had this:

```
INFO:root:Running Pip: '['C:\\Program Files\\Blender Foundation\\Blender 4.1\\4.1\\python\\bin\\python.exe', '-m', 'pip', 'install', 'tifffile==2023.4.12']'
INFO:root:Command succeeded: ['C:\\Program Files\\Blender Foundation\\Blender 4.1\\4.1\\python\\bin\\python.exe', '-m', 'pip', 'install', 'tifffile==2023.4.12']
INFO:root:stdout: Defaulting to user installation because normal site-packages is not writeable
Requirement already satisfied: tifffile==2023.4.12 in c:\users\jjohn\appdata\roaming\python\python311\site-packages (2023.4.12)
Requirement already satisfied: numpy in c:\program files\blender foundation\blender 4.1\4.1\python\lib\site-packages (from tifffile==2023.4.12) (1.24.3)
```

After restarting Blender it had worked!

**Comment by oanegros** [comment]
Thanks for the report! Blender 4.2 will introduce different package management, so hopefully this will fix it!

**Comment by oanegros** [comment]
fixed with including wheels in blender 4.2 extension management
